pyproject.toml

```diff
@@ -36,3 +36,4 @@ show_error_codes = true
 module = ["tests.*"]
 disallow_untyped_decorators = false
 disallow_untyped_defs = false
+warn_return_any = false
```

Comment:
NIP: Personally I prefer to use the `# type: ignore` comment instead of disabling this rule, because it often causes `Any` overuse

Reply:
That makes sense :+1:
Thanks for explanations